### PR TITLE
Update examples to 1.4.0

### PR DESCRIPTION
**Pull request by diarmidmackenzie**
This PR updates all examples to A-Frame 1.4.0, and fixes various issues arising:
- THREE.Math -> THREE.MathUtils
- Move from aframe-teleport-controls (old, unmaintained, broken in A-Frame 1.4.0) to blink-controls (maintained).
- A workaround for [A-Frame issue 5203](https://github.com/aframevr/aframe/issues/5203) (which looks like a regression in 1.4.0).
- Patch ammo-debug-drawer to addresses various THREE deprecations (would be nice to move to 1.0.1, but that uses ES6-style import/export, so needs PR #22 

Of the issues found upgrading to 1.4.0, the ones that aren't fixed are:
- #24, which I will handle as a separate PR.
- #25 which is specific to Cannon Worker Driver, and I don't plan on fixing.

The branch that includes this PR is temporarily published at:
https://diarmidmackenzie.github.io/aframe-physics-system/examples/

### CODE REVIEW

File: examples/ammo/constraints.html [hunk 1]
```diff
@@ -5,10 +5,10 @@
     <meta http-equiv="content-type" content="text/html; charset=utf-8">
     <meta name="viewport" content="width=device-width,initial-scale=1,shrink-to-fit=no,user-scalable=no,maximum-scale=1">
     <title>Examples • Constraints • AMMO</title>
-    <script src="https://aframe.io/releases/1.3.0/aframe.min.js"></script>
+    <script src="https://aframe.io/releases/1.4.0/aframe.min.js"></script>
     <script src="https://mixedreality.mozilla.org/ammo.js/builds/ammo.wasm.js"></script>
     <script src="https://unpkg.com/aframe-environment-component@1.3.1/dist/aframe-environment-component.min.js"></script>
-    <script src="https://cdn.jsdelivr.net/gh/fernandojsg/aframe-teleport-controls@master/dist/aframe-teleport-controls.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/aframe-blink-controls/dist/aframe-blink-controls.min.js"></script>
```
Comment (vincentfretin) [inline examples/ammo/constraints.html:11]: Better to use a release for blink-controls, sometimes jsdelivr serves you an old version with a url like this.
Use https://cdn.jsdelivr.net/npm/aframe-blink-controls@0.4.3/dist/aframe-blink-controls.min.js

Also update aframe-environment-component to 1.3.2

File: examples/cannon/constraints.html [hunk 2]
```diff
@@ -25,15 +25,18 @@
         view code
     </a>
     <a-scene environment>
-      <a-entity teleport-controls hand-controls="hand: left" static-body="shape: sphere; sphereRadius: 0.02;" grab></a-entity>
-      <a-entity teleport-controls hand-controls="hand: right" static-body="shape: sphere; sphereRadius: 0.02;" grab></a-entity>
-
-      <a-entity camera look-controls wasd-controls position="0 1.6 0">
+      <a-entity id="cameraRig"></a-entity>
```
Comment (vincentfretin) [inline examples/cannon/constraints.html:28]: I think the closing tag here is wrong.

**Review by vincentfretin — CHANGES_REQUESTED** [review]
Changes looks good, I added some comments. I just reviewed the changes in this PR, I didn't test the examples myself.

File: examples/ammo/constraints.html (hunk 1 above)
Comment (diarmidmackenzie) [reply examples/ammo/constraints.html]: Yes, good suggestion.

File: examples/cannon/constraints.html (hunk 2 above)
Comment (diarmidmackenzie) [reply examples/cannon/constraints.html]: Well spotted.

**Pull request merged**